Network.py

```diff
@@ -22,43 +37,74 @@ class graphType(Enum):
     TREE          = 8
 
 class Node:
-    nodeID = 0
+    """
+        Description: Node class represents a computer in the network. nodeID is used to identify
+                     the node, adjacentNodes is a list that stores a nodes directly connected neighbors.
+                     susceptibility var tells us if a node becomes susceptible to getting infectede
+                     from a virus.
+    """
+    nodeID = -1
     adjacentNodes = None
-    suceptibility = None
-    p_infected = None # probability to get infected by worm
+    susceptibility = None
     def __init__(self, nodeNum):
         self.nodeID = nodeNum
-        self.suceptibility = N.random.uniform(.0, .7)
+        self.susceptibility = N.random.uniform(.0, .7)
         self.adjacentNodes = []
-        self.p_infected = N.random.uniform(0, 0.5)
 
 
 class Network:
+    """
+        Description: Network class holds all nodes in the network. nodes is a dictionary
+                     which stores a Node object for each corresponding node number. infectedList
+                     is a list, the index of which represents that particular node and the value
+                     it holds (defined enums per the state class) tells us the current status
+                     of the node corresponding to that index. networkType holds a value that
+                     represents the type of network that is being created.
+                     createnetwork() method takes as input a string filename and passed it onto
+                     drawGraphFromFile() class which returns a list of pairs of numbers. Each pair
+                     of numbers represents a connection between those two nodes. The function uses each
+                     pair to construct a Node object and a dictionary item and insert them into the
+                     corresponding data structure.
+    """
     nodes = None
-    edges = None
     infectedList = None
     networkType = graphType.NONE
 
     def __init__(self, type):
-        self.edges = {}
+        self.nodes = {}

```

Comment:
edges was being used in the simulation file, and viruses file. Did you change it's name there too?